debian/rules

```diff
@@ -128,14 +135,15 @@ endif
 	# Delete private files from libraries so they don't get shipped in the -dev pacakges
 	rm -r $(TMP)/usr/include/mariadb/server/private
 
+	# Don't ship sql-bench at all, just delete it completely even though it builds
+	rm -r $(TMP)/usr/sql-bench
```

Comment:
I am not aware of any reasons why this is removed. Was there a discussion on this that I missed?

We should probably do a proper cmake fix to allow one to *not* build this if we don't want to ship it.
Then again, if we don't ship it, why not have it in a separate repository?
Any other input you can give me regarding this so I understand it better?